rdr/busybox/cephfs/app-busybox-2/resources/deployment/busybox-dc-1.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    appname: busybox_app2_cephfs-1
+    workloadpattern: simple_io
+  name: busybox-1
+spec:
+  replicas: 5
+  selector:
+    matchLabels:
+      workload: busybox-1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        workload: busybox-1
+    spec:
+      containers:
+      - command: ["/bin/sh", "-c"]
+        args:
+          - wget  https://raw.githubusercontent.com/red-hat-storage/ocs-workloads/master/rdr/busybox/scripts/simple_io.sh -O /mnt/test/simple_io.sh;
+            wget https://raw.githubusercontent.com/red-hat-storage/ocs-workloads/master/rdr/busybox/scripts/simple_io_integrity_check.sh  -O /mnt/test/simple_io_integrity_check.sh;
+            chmod +x /mnt/test/simple_io.sh; chmod +x /mnt/test/simple_io_integrity_check.sh;
+            /mnt/test/simple_io.sh
+        image: quay.io/ocsci/rdr-ocs-workload
+        env:
+        - name: IO_SCRIPT
+          value: "/mnt/test/simple_io.sh"
+        - name: DATA_INTEGRITY_SCRIPT
+          value: "/mnt/test/simple_io_integrity_check.sh"
+        imagePullPolicy: IfNotPresent
+        name: busybox
+        volumeMounts:
+        - mountPath: /mnt/test
+          name: mypvc
+      volumes:
+      - name: mypvc
+        persistentVolumeClaim:
+          claimName: busybox-pvc-1
```

Comment:
seems that all deployments use `busybox-pvc-1`

Reply:
Yes, it would be good for RWX volumes, WDYT?

Reply:
Sorry my bad fixing in new commit

Reply:
No, what I meant is this should be fine according to me

Reply:
If I get it right, each deployment controls 5 pods, and all those 5 use the same PVC.
Each deployment yaml should use a different PVC.

Reply:
Absolutely right

Reply:
That's how we wanted to do it for cephfs rwx volumes, WDYT about it?